prow/gerrit/adapter/adapter.go

```diff
@@ -253,9 +253,11 @@ func (c *Controller) processChange(latest client.LastSyncState, changeChan <-cha
 		now := time.Now()
 
 		result := client.ResultSuccess
-		if err := c.triggerJobs(log, instance, change); err != nil {
-			result = client.ResultError
-			log.WithError(err).Info("Failed to trigger jobs based on change")
+		if !c.shouldSkipProcessingChange(instance, change) {
```

Comment:
Please pass in and use `latest` rather than repeatedly calling `c.tracker.Current()`. That involves a lock and we don't need to be rereading this data, `latest` is still valid.

Reply:
Done.